### PR TITLE
Allow to display CJK characters

### PR DESCRIPTION
**Pull request by ubnt-intrepid**

### CODE REVIEW

File: source/app.d [hunk 1]
```diff
@@ -77,16 +107,17 @@ void print(ulong x, ulong y, string line, bool selected) {
...
 void updateQuery(bool init = false) {
-  string query = {
+  dstring query = {
     immutable prompt = "QUERY%s> %s";
     Appender!string query = appender!string;
 
     formattedWrite(query, prompt, E.matchByRegex ? "[regex]" : "[fuzzy]", E.query);
 
-    return query.data;
+    return query.data.to!dstring;
```
Comment (kubo39) [inline source/app.d]: Using `appnder!dstring` looks smarter!

File: source/app.d [hunk 2]
```diff
@@ -50,14 +50,44 @@ struct Env {
...
+
+  dstring s: a (decoded) string
+*/
+ulong mb_strwidth(dstring s)
+{
+  ulong len;
+  foreach (c; s) {
+    if (c < 0x0020) {
+    } else if (c >= 0x0020 && c < 0x2000) {
+      len +=1;
```
Comment (kubo39) [inline source/app.d]: (nit): space between `=` and `1`.

Comment (ubnt-intrepid) [reply source/app.d]: Thanks for find the mistake!

**Comment by kubo39** [comment]
@ys-nuem Thanks, LGTM.

**Comment by ubnt-intrepid** [comment]
Some fixes has not been finished.
I replaced the type of all string types to dstring.  It may change the behaviour of scoring algorithm.

**Pull request merged**